src/configs/unary-elementwise-config.c

```diff
@@ -1031,10 +1031,11 @@ static void init_f32_lrelu_config(void) {
     f32_lrelu_config.ukernel = (xnn_vunary_ukernel_fn) xnn_f32_vlrelu_ukernel__scalar_u4;
     f32_lrelu_config.init.f32_lrelu = xnn_init_f32_lrelu_scalar_params;
     f32_lrelu_config.element_tile = 4;
-  #elif XNN_ARCH_RISCV
-    f32_lrelu_config.ukernel = (xnn_vunary_ukernel_fn) xnn_f32_vlrelu_ukernel__scalar_u4;
+  #elif XNN_ARCH_RISCV && XNN_ENABLE_RISCV_VECTOR
+    const struct xnn_hardware_config* hardware_config = xnn_init_hardware_config();
+    f32_lrelu_config.ukernel = (xnn_vunary_ukernel_fn) xnn_f32_vlrelu_ukernel__rvv_u4v;
     f32_lrelu_config.init.f32_lrelu = xnn_init_f32_lrelu_scalar_params;
-    f32_lrelu_config.element_tile = 4;
+    f32_lrelu_config.element_tile = hardware_config->vlenb/sizeof(float)*4; // (VLENB/sizeof)*LMUL
```

Comment:
spaces around / and *
f32_lrelu_config.element_tile = hardware_config->vlenb * 4 / sizeof(float);

Reply:
done